autoload/LanguageClient.vim

```diff
@@ -1401,12 +1417,8 @@ endfunction
 
 function! LanguageClient_contextMenu() abort
     let l:options = keys(LanguageClient_contextMenuItems())
-
-    if get(g:, 'loaded_fzf') && get(g:, 'LanguageClient_fzfContextMenu', 1)
-        return fzf#run(fzf#wrap({
-                    \ 'source': l:options,
-                    \ 'sink': function('LanguageClient_handleContextMenuItem'),
-                    \ }))
+    if type(s:GetVar('LanguageClient_selectionUI')) is s:TYPE.funcref || s:GetVar('LanguageClient_selectionUI') ==? 'FZF'
```

Comment:
Same here, if the option is not set at all but FZF is loaded it will use the location list, but the current behaviour is different.